site/en/authors/author/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Author'
+description: ''
+permalink: '{{paged.permalink}}'
+layout: 'layouts/author-individual.njk'
+eleventyComputed:
+  title: '{{ paged.title | i18n(locale) or title }}'
```

Comment:
Will this fall back to 'Author' if there's no i18n version? (Again, I don't think we should translate author names, so maybe this is moot.)

Reply:
It should.